src/game/models/monsters.js

```diff
@@ -634,89 +634,88 @@ t . Dynamo
       tokens: ['dynamo'],
     },
   },
+
+ /* #<{(|//////////////////////////////////////////////////////////////////////////////
+ Deeps Below
+
+ w . Plague Staff
+ s . Unholy
+ t . Pandora's Box Shard
+ */
+    {
+     id: 'colossal_spider',
+     name: 'Colossal Spider',
+     str: 150,
+     ref: 130,
+     acc: 150,
+     con: 130,
+     flow: 150,
+     level: 200,
+     prizes: {
+       exp: 200,
+       items: ['Spider Leg'],
+       tokens: ['Pandoras Box'],
+    },
+},
+      id: 'urork',
+      name: 'Urork',
+      str: 150,
+      ref: 165,
+      acc: 150,
+      con: 165,
+      flow: 150,
+      level: 215,
+      prizes: {
+        exp: 220,
+        items: ['Shattered blade'],
+        tokens: ['Pandoras Box'],
+ },
+,
+ {
```

Comment:
Another bracket not using the proper indentation.

At this point I'm just pointing them out so you don't forget about them :)